lib/components/tag-chip/index.tsx

```diff
@@ -2,18 +2,24 @@ import React, { FunctionComponent } from 'react';
 import classNames from 'classnames';
 
 type OwnProps = {
-  onSelect: () => any;
+  onSelect?: (
+    event: React.MouseEvent<HTMLDivElement, globalThis.MouseEvent>
```

Comment:
what's `globalThis.MouseEvent` here? I haven't come across it in the React types yet

Reply:
it looks like it might be the default parameter so do we need it or could we also use `React.MouseEvent<HTMLDivElement>`?